static/package.json

```diff
@@ -36,7 +36,7 @@
     "locale-currency": "0.0.2",
     "mini-css-extract-plugin": "^1.6.0",
     "postcss": "^8.4.14",
-    "resolve-url-loader": "^3.1.4",
+    "resolve-url-loader": "^3.1.5",
```

Comment:
can we run `npm install` in the `static` directory to get the package-lock updates? also, might be worth running `npm audit fix` there too

Reply:
sure thing. I did try npm run audit, but there were no free fixes from that for the test-site